samples/oauth_ccg_postman_pre_request_script/CCG Pre-request Script.postman_collection.json

```diff
@@ -0,0 +1,216 @@
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							"const uuid = require(\"uuid\")",
+							"let kjur",
+							"",
+							"const aud = pm.collectionVariables.get(\"aud\")",
+							"const scope = pm.collectionVariables.get(\"scope\")",
+							"const clientId = pm.collectionVariables.get(\"clientId\")",
+							"const privatePem = pm.collectionVariables.get(\"privatePem\")",
+							"",
+							"function checkVariables() {",
+							"    const requiredVariables = [\"jsrsasign-js\", \"aud\", \"clientId\", \"privatePem\", \"scope\"]",
+							"    console.info(\"Checking if required collection variables are set...\")",
+							"    const missingOrNullOrUndefinedVariables = requiredVariables.filter(variable => {",
+							"        const value = pm.collectionVariables.get(variable)",
+							"        return value === \"\" || value === undefined || value === null",
+							"    })",
+							"    if (missingOrNullOrUndefinedVariables.length > 0) {",
+							"        missingOrNullOrUndefinedVariables.forEach(variable => {",
+							"            console.log(`Collection variable '${variable}' is either missing or has a null/undefined value.`)",
+							"        })",
+							"        return 0",
+							"    } else {",
+							"        console.info(\"All required collection variables are set with non-null values.\")",
+							"        return 1",
+							"    }",
+							"}",
+							"",
+							"function getAssertion() {",
+							"    const secondsSinceEpoch = Math.round(Date.now() / 1000)",
+							"    const header = {",
+							"        alg: \"RS256\",",
+							"        typ: \"JWT\"",
+							"    }",
+							"    const claims = {",
+							"        \"aud\": aud,",
+							"        \"iss\": clientId,",
+							"        \"sub\": clientId,",
+							"        \"jti\": uuid.v4(),",
+							"        \"iat\": secondsSinceEpoch,",
+							"        \"exp\": secondsSinceEpoch + 60",
+							"    }",
+							"    const jwt = kjur.jws.JWS.sign(null, header, claims, privatePem)",
+							"    console.info(\"JWT assertion: \" + jwt)",
+							"    const urlencoded = [{",
+							"            key: \"grant_type\",",
+							"            value: \"client_credentials\",",
+							"            disabled: false",
+							"        },",
+							"        {",
+							"            key: \"client_assertion_type\",",
+							"            value: \"urn:ietf:params:oauth:client-assertion-type:jwt-bearer\",",
+							"            disabled: false",
+							"        },",
+							"        {",
+							"            key: \"client_assertion\",",
+							"            value: jwt,",
+							"            disabled: false",
+							"        },",
+							"        {",
+							"            key: \"scope\",",
+							"            value: scope,",
+							"            disabled: false",
+							"        }",
+							"    ]",
+							"    return urlencoded",
+							"}",
+							"",
+							"function getToken(urlencoded) {",
+							"    const req = {",
+							"        url: aud,",
```

Comment:
This needs to be the OAuth Proxy branded `aud` to work for API's utilizing launch context.